artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fartifacttemplates/Shor-UI_connectTo-w1/files/connectTo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Parameters:
+# $Port
+# $VMIP
+# $ContainerIP
+# $SOURCE_AppName
+
+if [[ ! -z $VMIP ]]; then
+    sed -i "s/VMIP/$VMIP/g" /var/www/html/$SOURCE_AppName/index.html
+elif [[ ! -z ContainerIP ]]; then
+    sed -i "s/VMIP/$ContainerIP/g" /var/www/html/$SOURCE_AppName/index.html
```

Comment:
Should be $ContainerIP in the if part right?

Reply:
Nope, since the placeholder in the files is called `VMIP`

Reply:
[[ ! -z ContainerIP ]]